qarakusi.am.py

```diff
@@ -0,0 +1,32 @@
+#!/usr/bin/env python3
```

Comment:
Is it possible to do all this in Makefile? Or at least run this from Makefile someway.
Also, I'm not sure if `/usr/bin/env python3` will work on Windows.

Reply:
I'm not fully on board with how we use a Makefile in this project. The fact that all of the target in the Makefile are marked as `.PHONY` is a good indication that we aren't making a very idiomatic use of `make`. However, I'm happy to have that conversation separately.

To alleviate your concerns on platform compatibility with Windows, I ran the script in 3 shells under Windows:

1. cmd.exe:
![image](https://user-images.githubusercontent.com/619878/170186410-a0db5c52-f84f-47bd-8a18-407f2497a690.png)

2. PowerShell:
![image](https://user-images.githubusercontent.com/619878/170186467-7c5f863a-d583-48ab-ace2-c4a13a929f89.png)

3. zsh running in ConEmu:
![image](https://user-images.githubusercontent.com/619878/170186555-e2079ad0-2e36-42a0-86b3-1c59f4d00205.png)

Reply:
Great, thanks!